conda/recipes/rmm/meta.yaml

```diff
@@ -40,22 +41,36 @@ requirements:
     - ninja
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - {{ compiler('cuda') }} {{ cuda_version }}
+    {% if cuda_major == "11" %}
+    - {{ compiler('cuda11') }} {{ cuda_version }}
+    {% else %}
+    - cuda-version {{ cuda_version }}
+    - {{ compiler('cuda') }}
+    {% endif %}
     - sysroot_{{ target_platform }} {{ sysroot_version }}
   host:
-    - cuda-python >=11.7.1,<12.0
+    {% if cuda_major == "11" %}
     - cudatoolkit ={{ cuda_version }}
+    - cuda-python >=11.7.1,<12.0
+    {% else %}
+    - cuda-python >=12.0,<13.0
```

Comment:
Not a blocker: Could we use `{{ cuda_spec }}` here instead for both? I guess though if there is a minimum on `11.7.1` for CUDA 11, maybe we do want to be explicit about it instead of `>=11`?

Also, if worried about alpha versions, the upper bounds should be suffixed with `a0` (ie `<13.0a0`).

Reply:
Good call on alphas.

I don't think we can rely on this matching `{{ cuda_spec }}`. Historically we've needed to use newer cuda-python versions for older toolkits with CUDA 11 and I suspect we'll need the same for CUDA 12 at some point due to bug fixes or packaging changes.

Reply:
Building on Ray's suggestion, had some ideas to simplify this further

Reply:
Appears no longer needed as `cuda-python`'s `run_exports` is relied on. Marking resolved